src/tools/perf/libperf.c

```diff
@@ -1115,53 +1117,55 @@ ucs_status_t ucx_perf_run(ucx_perf_params_t *params, ucx_perf_result_t *result)
 
 
 /* multiple threads sharing the same worker/iface */
+#include <omp.h>
```

Comment:
what happens if omp.h is not present?
why is it included in the middle of the file?